app/tip/page.tsx

```diff
@@ -426,20 +426,20 @@ export default function TipPage() {
                 </div>
               </form>
             </Form>
-            <div className="text-center flex flex-col gap-1">
+            {/* <div className="text-center flex flex-col gap-1">
               <p className="items-center my-2">or</p>
 
               <DooglyTippingButton
                 key={dooglyRecipient}
-                buttonText="Tip with Doogly (beta)"
+                buttonText="Tip with Doogly"
                 modalTitle="Make a Donation"
                 config={{
                   destinationChain: "celo",
                   destinationAddress:
-                    "0xFa1aD6310C6540c5430F9ddA657FCE4BdbF1f4df", // Doogly Gateway on Celo
-                  receiverAddress: dooglyRecipient, // Tip recipient
+                    "0xFa1aD6310C6540c5430F9ddA657FCE4BdbF1f4df",
```

Comment:
I think this would be better put in a constant instead of removing the comments, but it's ok